WcaOnRails/config/locales/en.yml

```diff
@@ -405,3 +405,16 @@ en:
       country: "Country"
       podiums: "Podiums"
       no_persons_found: "No person matches the given criteria."
+  committees:
+    new:
+      create_committee: "Create New Committee"
+    committee_form:
+      supports_md_html: "Supports <a href='https://daringfireball.net/projects/markdown/basics' target='_blank'>Markdown</a>"
+      name_html: "The name of the committee (e.g., Delegates, or Software)"
+      email_html: "The email of the committee (e.g., software@worldcubeassociation.org)"
+      slug_html: "This will be used as part of the public URL for this committee."
+      duties_html: "What this committee is responsible for? %{md}"
```

Comment:
As @jonatanklosko said these should be in `simple_form.hints`, but this last one will probably stay here.
Also note that the suffix `_html` just tells rails that the text is html safe and should not be escaped, if the text doesn't contain html you can drop it :)